tf-modules/report-executions/variables.tf

```diff
@@ -0,0 +1,38 @@
+
+variable "prefix" {
+  type = string
+}
+
+variable "aws_profile" {
+  type    = string
+  default = "default"
+}
+
+variable "aws_region" {
+  type    = string
+  default = "default"
+}
+
+variable "executions_table" {
+  type    = string
+  default = "default"
+}
+
+variable "permissions_boundary" {
+  type = string
+}
+
+variable "vpc_id" {
+  type    = string
+  default = ""
+}
+
+variable "subnet_ids" {
+  type    = list(string)
+  default = []
+}
+
+variable "security_groups" {
+  type    = list(string)
+  default = []
+}
```

Comment:
I like Marc's convention for the Variables file: group the variables with no defaults in a section marked `# Required` and the ones with defaults in a section marked `# Optional`. Can we do that here?